subsys/dsp/Kconfig

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2022 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+config DSP_BACKEND_HAS_STATIC
+	bool
+
+config DSP_BACKEND_CMSIS
+	bool "Use the CMSIS-DSP library as the math backend"
+	depends on CMSIS_DSP
+	default y if CMSIS_DSP
+	select DSP_BACKEND_HAS_STATIC
+	help
+	  Implement the various zephyr DSP functions using the CMSIS-DSP library. This feature
+	  requires the CMSIS module to be selected.
```

Comment:
Please fix the "no new line at the end of the file."

Reply:
Done